CHANGELOG.md

```diff
@@ -4,6 +4,120 @@ All notable changes to Router will be documented in this file.
 
 This project adheres to [Semantic Versioning v2.0.0](https://semver.org/spec/v2.0.0.html).
 
+# [1.19.0] - 2023-05-19
+
+## 🚀 Features
+
+### `require_authentication` option to reject unauthenticated requests ([Issue #2866](https://github.com/apollographql/router/issues/2866))
+
+While the authentication plugin validates queries with JWT, it does not reject unauthenticated requests, and leaves that to other layers. This allows co-processors to handle other authentication methods, and plugins at later layers to authorize the reqsuest or not. Typically, [this was done in rhai](https://www.apollographql.com/docs/router/configuration/authn-jwt#example-rejecting-unauthenticated-requests).
+
+This now adds an option to the Router's YAML configuration to reject unauthenticated requests. It can be used as follows:
+
+```yaml
+authorization:
+	require_authentication: true
+```
+
+The plugin will check for the presence of the `apollo_authentication::JWT::claims` key in the request context as proof that the request is authenticated.
+
+
+By [@Geal](https://github.com/Geal) in https://github.com/apollographql/router/pull/3002
+
+## 🐛 Fixes
+
+### prevent span attributes from being formatted to write logs 
+
+we do not show span attributes in our logs, but the log formatter still spends some time formatting them to a string, even when there will be no logs written for the trace. This adds the `NullFieldFormatter` that entirely avoids formatting the attributes
+
+By [@Geal](https://github.com/Geal) in https://github.com/apollographql/router/pull/2890
+
+## 🐛 Fixes
+
```

Comment:
No idea why there are two _Fixes_ sections.  That must be a bug in the xtask.  I've never seen that before!  I'm _guessing_ it was my filename for the 2.4.5 release.

```suggestion
## 🐛 Fixes

### Prevent span attributes from being formatted to write logs 

we do not show span attributes in our logs, but the log formatter still spends some time formatting them to a string, even when there will be no logs written for the trace. This adds the `NullFieldFormatter` that entirely avoids formatting the attributes

By [@Geal](https://github.com/Geal) in https://github.com/apollographql/router/pull/2890

```